global.json

```diff
@@ -0,0 +1,5 @@
+{
+  "sdk": {
+    "version": "2.0.0"
```

Comment:
I reckon this raises the barrier to entry and should be left to your build script

Reply:
I agree, but as I just discovered a moment ago appveyor fails to run the tests without it, not sure how to get around that.